nilearn/_utils/niimg.py

```diff
@@ -0,0 +1,157 @@
+"""
+Neuroimaging file input and output.
+"""
+# Author: Gael Varoquaux, Alexandre Abraham, Philippe Gervais
+# License: simplified BSD
+
+import copy
+import gc
+import collections
+from six import string_types
+
+import numpy as np
+import nibabel
+
+from .numpy_conversions import as_ndarray
+
+
+def _safe_get_data(img):
+    """ Get the data in the image without having a side effect on the
```

Comment:
> OK. The official nibabel way to do that is np.array(img.dataobj).

I don't think that it was possible when we wrote this code. I remember
having a lengthy discussion with Matthew Brett about this.